modules/rosa-list-objects.adoc

```diff
@@ -0,0 +1,534 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-list-objects_{context}"]
+= List and describe objects
+
+[role="_abstract"]
+This section describes the `list` and `describe` commands for clusters and resources.
+
+[id="rosa-list-oaddon_{context}"]
+== list addon
+
+Lists the managed service add-on installations.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list addons --cluster=<cluster_name>
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to list the add-ons for.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+[id="rosa-list-clusters_{context}"]
+== list clusters
+
+Lists all of your clusters.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list clusters [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--count
+|The number (integer) of clusters to display. Default: `100`
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+[id="rosa-list-idps_{context}"]
+== list idps
+
+Lists all identity providers (IDPs) for a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list idps [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster that the IDPs will be listed for.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Example
+List all identity providers (IDPs) for a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa list idps --cluster=mycluster
+----
+
+[id="rosa-list-ingresses_{context}"]
+== list ingresses
+
+Lists all API and ingress endpoints for a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list ingresses --cluster=mycluster [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster that the IDPs will be listed for.
```

Comment:
Required: 
(Replace throughout this doc set)

Reply:
Global search and replace done.